src/thing_data.h

```diff
@@ -181,7 +181,7 @@ struct Thing {
         unsigned char byte_13;
         unsigned char byte_14;
         unsigned char byte_15;
-        unsigned char byte_16;
+        unsigned char hit_type;
```

Comment:
I think it should be new `struct effect { ... }` above and field should be used as `efftng->effect.hit_type`

Reply:
I don't quite understand what you mean here.

Reply:
`struct Thing` contains a large union of different fields. All these byte_13 byte_13a etc. are from them.
Actually each that struct is an enum branch defined by object type and model.
So we may assume that there is a "effect" branch with same layout of fields where `byte_16` means `hit_type` and that branch should be chosen when it is an effect. Same for "shoot" branch.

Reply:
I looked and I believe this particular one is related to trapdoors, not effects.